### PR TITLE
feat(STONEINTG-533): add 2 panels for SLO #3

### PR DESCRIPTION
**Pull request by jencull**
Panel one Latency of release creation
Panel two violations of the 90% in 10 seconds slo

## Maintainers will complete the following section

- [x] Commit messages are descriptive enough ([hints](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/))
- [ ] Code coverage from testing does not decrease and new code is covered ([check the PR coverage on codecov](https://app.codecov.io/gh/redhat-appstudio/integration-service/pulls))
- [ ] [Controllers diagrams](https://github.com/redhat-appstudio/integration-service/tree/main/docs) are updated when PR changes controllers code  (if applicable)

### CODE REVIEW

**Comment by MartinBasti** [comment]
a screenshot would be nice, I'm still not good at visualizations how dahsboard looks from JSON :)

File: config/grafana/dashboards/integration-service-dashboard.json [hunk 1]
```diff
@@ -430,6 +430,203 @@
...
+    {
+      "id": 23,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 9
+      },
+      "type": "graph",
+      "title": "#3 - Latency  of Release Creation",
```
Comment (MartinBasti) [inline config/grafana/dashboards/integration-service-dashboard.json:442]:
```suggestion
      "title": "#3 Latency  of Release Creation",
```

Comment (jencull) [reply config/grafana/dashboards/integration-service-dashboard.json]: corrected. second instance corrected also.

**Pull request merged**